subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/configuration/GradleInstrumentationPluginConfiguration.kt

```diff
@@ -0,0 +1,104 @@
+package com.avito.instrumentation.configuration
+
+import com.avito.report.model.Team
+import com.avito.slack.model.SlackChannel
+import com.google.common.annotations.VisibleForTesting
+import groovy.lang.Closure
+import org.gradle.api.NamedDomainObjectContainer
+import org.gradle.api.NamedDomainObjectSet
+import org.gradle.api.Project
+import java.io.Serializable
+
+object InstrumentationPluginConfiguration {
```

Comment:
I tried to delete not needed wrapper type. But can't because of backward compatibility